webpack.config.js

```diff
@@ -1,17 +1,79 @@
 const path = require("path");
 const webpack = require('webpack');
+const HTMLWebpackPlugin = require('html-webpack-plugin');
+const CompressionPlugin = require("compression-webpack-plugin");
+const SWPrecacheWebpackPlugin = require('sw-precache-webpack-plugin');
 
-const config = {
-  entry: {
-    app: [
-      './src/index.js'
-    ]
-  },
 
+
+
+//enviroment variable
+const DEVELOPMENT = process.env.NODE_ENV === 'development';
+const PRODUCTION = process.env.NODE_ENV === 'production';
+const TEST = process.env.NODE_ENV === 'test'
+
+const entry = PRODUCTION
```

Comment:
isn't production and development config suppose to be different files or something?